src/dongyeon-lee/230209/P92341.kt

```diff
@@ -0,0 +1,37 @@
+package `dongyeon-lee`.`230209`
+
+import kotlin.math.ceil
+
+class Solution {
+    fun solution(fees: IntArray, records: Array<String>): List<Int> {
+        val (dTime, dFee, pTime, pFee) = fees
+        val lastOfDay = ("23:59").toMinutes()
+        val parkingMap = hashMapOf<String, Int>() // 차량 번호, 시각
+        val feeMap = hashMapOf<String, Int>()
+
+        records.forEach {
+            val  (t, num, _) = it.split(" ")
+            val time = t.toMinutes()
+            if (parkingMap.containsKey(num)) {
+                val diff = time -parkingMap.getOrDefault(num, 0)
```

Comment:
```suggestion
                val diff = time - parkingMap.getOrDefault(num, 0)
```
컨벤션을 지켜주세요!

Reply:
```suggestion
                val diff = time - parkingMap.getValue(num)
```
어차피 위에서 containsKey로 검사했는데
getOrDefault 쓸 이유가 없네요